src/utils/urls.js

```diff
@@ -34,18 +34,25 @@ function normalize (url) {
  *
  * The returned URL does not include the query search params (i.e. excludes ?redirectTo=/hub/login).
  *
+ * The parameter `baseOnly` can be used to define whether the final URL will incl
```

Comment:
Is it OK to use camel case @hjoliver ? Sorry my OCD :laughing:

Reply:
Yes, much as I hate it, I should conform to the practice here!